src/main/java/util/JavaVersion.java

```diff
@@ -0,0 +1,92 @@
+package util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Represents Java version
+ */
+public class JavaVersion implements Comparable<JavaVersion> {
+
+    private static final Pattern JAVA_8_VERSION_PATTERN =
+            Pattern.compile("(\\d+)\\.(\\d+)\\.(\\d+)\\_(\\d+)(-b(\\d+))?");
+
+    public static final String WARNING_MSG_OUTDATED_JAVA_VERSION =
```

Comment:
I don't think this belongs in this class. JavaVersion is a general utility class. It should not contain a HT-specific message shown to the user.